src/oed.wppl

```diff
@@ -91,15 +95,14 @@ var EIG = function(args) {
                 var scoreY = m(x,y);
                 factor(scoreY);
             }
-            // TODO: just use AIG() here?
-            var mPosterior = inferM2(function() {
-                var m2Name = mNameSample(), m2 = mFuncs[m2Name];
-                var ll = m2(x,y);
-                factor(ll);
-                return m2Name
-            });
-            var kl = KL(mPosterior, mPrior);
-            return kl;
+
+            return AIG({
```

Comment:
This is a little inefficient -- we've already constructed the prior over models inside `EIG` but by calling `AIG` as a subroutine, we're going to recompute the prior for every `x` and `y`.

Can you roll back this part and we'll just merge in the AIG infer change?